workspaces/util/src/getLines.ts

```diff
@@ -1,5 +1,7 @@
 export function* getLines(text: string): Generator<string, void, void> {
-  for (const [line] of text.matchAll(/[^\n]+\n?|\n/g)) {
+  for (const [line] of text.matchAll(
+    /[^\n\v\f\r]+(\n|\r|\v|\f)?|(\n|\r|\v|\f)/g,
```

Comment:
Hey, thanks for thinking about different kinds of new lines. I think this is actually not entirely correct, because it won't do the right thing for Windows. On Windows new lines are usually not just one character, but two, specifically "\r\n". So "hello\r\nworld\r\n" is two lines, but I think this regex will break it into "hello\r", "\n", "world\r", and "\n" when we just wanted "hello\r\n" and "world\r\n". It might even be nice to have a test case testing exactly this.

Let's stick to only handling \n for now, using the old regex. If a need arises to handle other kinds of new lines, we can update it later.